1/assignment-week1.py

```diff
@@ -0,0 +1,83 @@
+print ("Week 1 Assignment - Math Equation Evaluator")
+print ("Intern - Shu Wei")
+print ("")
+
+eArray = list(map(str, input("Please enter your input with spaces between them: ").split()))
+
+#Definitions
+aCount = (len(eArray))
+fCount = 3
+nCount = 4
+needle = 0
+
+# Check whether the first two inputs are numbers and the last input is a sign
+try:
+    check1 = int(eArray[0])
+except ValueError:
+    print("Please make sure that the input #1 is a number")
+    exit()
+try:
+    check2 = int(eArray[1])
+except ValueError:
+    print("Please make sure that the input #2 input is a number")
+    exit()
+
+if aCount > 3:
+    CheckNCount = 4
+    while (CheckNCount <= aCount):
+        try:
+            checkN = int(eArray[CheckNCount-1])
+        except ValueError:
+            print ("Please make sure that the input #", CheckNCount, "is a number")
+            exit()

```

Comment:
`exit` are extremely bad practice. avoid at all cost

Reply:
What should I use instead to make it stop running?

Reply:
This whole thing should be in a function and you can return from the function. Or if you're expecting the validation to work, you can just throw an error and let the program crash